specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -1924,7 +1996,7 @@
             ]
           }
         },
-        "labelValue": {
+        "value": {
```

Comment:
You will need to add "x-ms-client-name": "labelValue" so that the SDKs don't have breaking changes.

Reply:
Addressed